db/migrate/20181102145252_add_regions_and_subscriptions.rb

```diff
@@ -0,0 +1,55 @@
+class AddRegionsAndSubscriptions < ActiveRecord::Migration[5.1]
+  def change
+    create_table "subscriptions", :id => :bigserial do |t|
+      t.references "source", :type => :bigint, :index => true, :null => false
+      t.string     "source_ref"
+      t.string     "name"
+      t.timestamps
+      t.datetime   "archived_on", :index => true
+      t.index      %i(source_id source_ref), :unique => true
+    end
+
+    add_foreign_key :subscriptions, :sources, on_delete: :cascade
+
+    create_table "regions", :id => :bigserial do |t|
```

Comment:
Can we come up with an alternate name?  I'm concerned if/when we bring this back into ManageIQ that this will be a huge problem.

Reply:
@Fryguy @agrare ```source_regions``` maybe?

Reply:
:+1:

Reply:
and I would guess ```source_subscriptions``` so we keep it consistent?